doc/test_organization/managing_tests_dependencies.qbk

```diff
@@ -7,17 +7,31 @@
 
 [section:tests_dependencies Managing test dependencies]
 
-Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators]. 
+In general, you should make sure that test cases are independent on one another.
```

Comment:
Because test case independence is corner stone of TDD

On Fri, May 22, 2015 at 5:51 PM, Raffi Enficiaud <notifications@github.com>
wrote:

> In doc/test_organization/managing_tests_dependencies.qbk
> <https://github.com/boostorg/test/pull/64#discussion_r30933423>:
>
> > @@ -7,17 +7,31 @@
> >
> >  [section:tests_dependencies Managing test dependencies]
> >
> > -Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators].
> > +In general, you should make sure that test cases are independent on one another.
>
> why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/boostorg/test/pull/64/files#r30933423>.
>



-- 
Gennadiy Rozental